spec/controllers/pxe_controller/iso_datastores_spec.rb

```diff
@@ -0,0 +1,15 @@
+describe PxeController do
+  before(:each) do
+    set_user_privileges
+  end
+
+  describe "#iso_datastore_set_form_vars" do
+    it 'assigns form options' do
+      ems = FactoryGirl.create(:ems_redhat)
+      controller.instance_variable_set(:@isd, FactoryGirl.create(:iso_datastore, ems_id: ems))
+      controller.send(:iso_datastore_set_form_vars)
+
+      expect(assigns(:edit)[:emses]).to eq([[ems.name, ems.id]])
```

Comment:
Yeah, I think last week I wrote it that way, then forgot and changed it this morning with your last comment. I'm rewriting it to be clear and cover both cases.